dydx/dydxPresenters/dydxPresenters/_v4/Profile/Notifications/dydxNotificationsSettingsViewBuilder.swift

```diff
@@ -57,7 +65,10 @@ private class dydxNotificationsSettingsViewPresenter: SettingsViewPresenter {
                                                      cancelTitle: DataLocalizer.shared?.localize(path: "APP.GENERAL.CANCEL", params: nil) ?? "Cancel"
             )
         }
-        // sync the settings with the system permission
+        reloadSettings()
```

Comment:
would this get called while user is being prompted to turn on notifications? Does this have an effect?

Reply:
Yeah.. it gets called, which sets the value back... But when the system notification value changes, it will be synced again.